### DISCUSSION TITLE
SONiC  202505 Release Proposal

### DISCUSSION THREAD

**Discussion by zhangyanzhao** [Azure/SONiC#1843] [General]
Create this thread to discuss SONiC 202505 release proposal. The release plan will be captured at https://github.com/orgs/sonic-net/projects/25/views/1. This thread is for community to share some ideas for that release. Please leave your comments to this thread.

**Comment by tsmjay** [comment]
I would like to push for better IPv6 support in SONiC 202505, specifically OSPFv3.

**Comment by bradh352** [comment]
I would like to contribute PRs to upgrade libyang in SONiC for this release as mentioned on the mailing list: https://lists.sonicfoundation.dev/g/sonic-dev/message/938

First PR in the series is being formed here: https://github.com/sonic-net/sonic-buildimage/pull/21679

I have sonic-yang-models ready to go, and sonic-yang-mgmt mostly finished but no PRs submitted for those.  I will also need to patch sonic-utilities and swss-common, so there will need to be tight coordination on merging PRs.

**Comment by nakano-omw** [comment]
@zhangyanzhao 

I am writing to request the inclusion of two function in the upcoming SONiC 202505 release. Could you please add the following HLDs?
https://github.com/orgs/sonic-net/projects/25

・OpenConfig Platform Transceiver HLD https://github.com/sonic-net/SONiC/pull/1858
・Fpmsyncd Next Hop Table Enhancement HLD  https://github.com/sonic-net/SONiC/pull/1425

If you need any further details or have any questions, please feel free to contact me.

**Reply by zhangyanzhao** [reply]
Added

**Comment by mbze430** [comment]
I still want to know what happened to MSTP and IGMP Snooping things that have been "in the work" for years now

**Reply by vaoneiler** [reply]
Hi did you get a response on this?

**Comment by vaoneiler** [comment]
who is responding to questions, suggestions or other inquiries?
